Makefile

```diff
@@ -0,0 +1,16 @@
+lint: bandit black flake8 isort pylama
+
+bandit:
+	pipenv run bandit -r asaps
+
+black:
+	pipenv run black --check --diff .
+
+flake8:
+	pipenv run flake8 .
+
+isort:
+	pipenv run isort . --diff
+
+pylama:
```

Comment:
Add `make test` command per comment above, and I think we should just remove Pylama from this repo if we're not using it. You can still install and run it locally for more info, but right now including it in the code isn't adding anything.